Common/GPU/OpenGL/thin3d_gl.cpp

```diff
@@ -406,15 +406,16 @@ class OpenGLContext : public DrawContext {
 		stencilWriteMask_ = writeMask;
 		stencilCompareMask_ = compareMask;
 		// Do we need to update on the fly here?
+		const auto depthStencil = curPipeline_->depthStencil;
```

Comment:
I think whenever it's not too hard, let's be explicit about the type for these instead of using auto. Clearer and doesn't really take up a lot of code space.